packages/components/package.json

```diff
@@ -21,6 +21,7 @@
         "@suite-common/icons": "workspace:*",
         "@suite-common/suite-constants": "workspace:*",
         "@suite-common/suite-utils": "workspace:*",
+        "@suite-common/token-definitions": "workspace:*",
```

Comment:
This is going to include redux, jws and other non-ui libs into components package